vyper/venom/venom_to_assembly.py

```diff
@@ -147,6 +147,14 @@ def generate_evm(self, no_optimize: bool = False) -> list[str]:
         asm: list[Any] = []
         top_asm = asm
 
+        def bb_cleanup_needed(bb: IRBasicBlock) -> bool:
```

Comment:
kind of weird for this to be an inline function. maybe it should be a function in `vyper/venom/basicblock.py` or `vyper/venom/function.py`?